shared/oae/api/oae.api.js

```diff
@@ -212,6 +213,12 @@ define(['underscore', 'oae.api.admin', 'oae.api.authentication', 'oae.api.config
          * @param  {Function}       callback            Invoked when the accept invitation request is complete
          */
         var acceptInvitation = function(callback) {
+            // We cannot accept an invitation as the anonymous user. Let the
+            // signup page handle it
+            if (oae.data.me.anon) {
```

Comment:
Incoming invitations are sent to the `/signup` page, which handles this specially. There should be no case to be on `/`, for example, and be anonymous with invitation data in the query string.